specs/fulu/beacon-chain.md

```diff
@@ -151,6 +167,64 @@ class BeaconState(Container):
 
 ### Misc
 
+#### New `BlobScheduleEntry`
+
+```python
+@dataclass
+class BlobScheduleEntry(object):
```

Comment:
Where is this type being used?

Reply:
Good call out. This was used, no longer is. We can remove it.

Reply:
Resolved in https://github.com/ethereum/consensus-specs/commit/b00d6a179ad307659a1fcdf26be4bb3d73c27a29.